doc/source/reference/arrays.dtypes.rst

```diff
@@ -178,7 +178,7 @@ Built-in Python types
     :class:`bool`     :class:`bool\_`
     :class:`float`    :class:`float\_`
     :class:`complex`  :class:`cfloat`
-    :class:`str`      :class:`string`
+    :class:`str`      :class:`bytes` (Python2) or :class:`unicode\_` (Python3)
```

Comment:
I think it's important to include `str` in this table, given how common it is to see `dtype=str`.